src/webviews/apps/plus/shared/components/home-account-content.ts

```diff
@@ -219,6 +219,13 @@ export class GLHomeAccountContent extends LitElement {
 					this.hasAccount,
 					() => html`
 						<span class="header__actions">
+							<gl-button
+								appearance="toolbar"
+								href="command:gitlens.views.home.account.refresh"
+								tooltip="Refresh"
+								aria-label="Refresh"
+								><code-icon icon="refresh"></code-icon
+							></gl-button>
```

Comment:
Should we think of this as a "refresh" or a "re-sync"? @justinrobots 

If "refresh" then this is good, but I'd change the tooltip to "Refresh Account Status"
If "re-sync" then maybe, use the sync icon and the tooltip to be "Synchronize Account Status"

Reply:
Sync does seem _a little_ more concise and it would add a better distinction from the view's refresh.